modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java

```diff
@@ -0,0 +1,61 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.LicenseAcceptance;
+
+/**
+ * @author Stefan Hufschmidt
+ */
+public class MSSQLServerContainer<SELF extends MSSQLServerContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
+    static final String NAME = "mssqlserver";
+    static final String IMAGE = "microsoft/mssql-server-linux";
+    public static final Integer MS_SQL_SERVER_PORT = 1433;
+    private String username = "SA";
+    private String password = "A_Str0ng_Required_Password";
+
+    public MSSQLServerContainer() {
+        this(IMAGE + ":latest");
+        LicenseAcceptance.assertLicenseAccepted(IMAGE);
```

Comment:
I think we should move this line right before `addEnv("ACCEPT_EULA", "Y")` to make it even more explicit and also avoid ignoring the file when user passes a version with `MSSQLServerContainer("microsoft/mssql-server-linux:1.2.3")` (the check is not included in `MSSQLServerContainer(final String dockerImageName)` constructor)